src/T2DMap.cpp

```diff
@@ -707,7 +707,7 @@ void T2DMap::paintEvent(QPaintEvent* e)
                 continue;
             }
             if (mapLabel.text.length() < 1) {
-                mpMap->mapLabels[mAreaID][it.key()].text = "no text";
+                mpMap->mapLabels[mAreaID][it.key()].text = tr("no text", "Default text if a label is created in mapper with no text");
```

Comment:
I have a suspicion that this is an internal string and is not shown to the user - so should be a `QStringLiteral` or even as there is a `QLatin1String` overload for `QString::operator==`, a `QLatin1String` instead of a `tr`

Reply:
It is shown to users *sometimes*: 
![image](https://user-images.githubusercontent.com/117238/47624627-cbd47380-db1e-11e8-8cb1-41e2953b21c3.png)

Reply:
:-1: Damn and blast....